gordo_client/io.py

```diff
@@ -78,11 +74,11 @@ def _handle_response(resp: requests.Response, resource_name: Optional[str] = Non
         In case of network or IO errors
     """
     if 200 <= resp.status_code <= 299:
-        if _is_json_response(resp):
+        if _is_model_response(resp):
+            return resp.content
+        elif _is_json_response(resp):
             return resp.json()
-        elif _is_azure_login_response(resp):
-            raise BadGordoRequest(f"Azure login page response found while fetching resource: {resource_name}.")
-        return resp.content
+        raise BadGordoResponse(f"Bad gordo response found while fetching resource: {resource_name}.")
```

Comment:
```suggestion
        resource_msg =  f"while fetching resource: {resource_name}" if resource_name else ""
        raise BadGordoResponse(f"Bad gordo response found{resource_msg}.", resp.content)
```
In this case, we will be able to check if the content contains an authorization string